dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/AssignTrn/Trn.cshtml.cs

```diff
@@ -25,7 +25,7 @@ public TrnModel(TeacherIdentityServerDbContext dbContext, IDqtApiClient dqtApiCl
     [BindProperty]
     [Display(Name = "What is their teacher reference number (TRN)?")]
     [Required(ErrorMessage = "Enter a TRN")]
-    [RegularExpression(@"^\d{7}$", ErrorMessage = "TRN must be 7 digits")]
+    [RegularExpression(@"\A\D*(\d{1}\D*){7}\D*\Z", ErrorMessage = "TRN must be 7 digits")]
```

Comment:
Updated regex to same as in Trn/HasTrnPage

Reply:
I think we want the simpler regex here (the support staff should know to always use a numeric form of the TRN).

Reply:
Ah ok didn't realise they could always be represented as 7 digits